test/E2ETests/E2ETests/Tables/TablesEndToEndTests.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.Azure.Documents;
+using Microsoft.Azure.Functions.Worker.E2ETests.Helpers;
+using Newtonsoft.Json;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Azure.Functions.Tests.E2ETests.Tables
+{
+    [Collection(Constants.FunctionAppCollectionName)]
+    public class TablesEndToEndTests : IDisposable
+    {
+        private readonly IDisposable _disposeLog;
+        private FunctionAppFixture _fixture;
+        private const string partitionKey = "Partition";
+        private const string firstRowKey = "FirstRowKey";
+        private const string firstValue = "FirstValue";
+        private const string secondRowKey = "SecondRowKey";
+        private const string secondValue = "SecondValue";
+        private const string thirdRowKey = "ThirdRowKey";
+        private const string thirdValue = "ThirdValue";
```

Comment:
nit: I would prefer if we define expected values that we are asserting against in the test method that they are used in. It's okay to be repetitive in tests. It makes tests easier to follow and safer in a sense as we have all the test items self contained